cypress/integration/e2e.js

```diff
@@ -0,0 +1,58 @@
+it('Behaves as desired', () => {
+  cy.visit('http://localhost:3000/');
+
+  cy.contains('Title').type('Hello');
+  cy.contains('Description').type('Really long text.');
+  cy.get('button').contains('Add').click();
+
+  cy.get('h3').contains('Hello').should('exist');
+  cy.get('p').contains('Really long text.').should('exist');
+
+  cy.get('#calendar')
+    .get('button.bg-indigo-700')
+    .get('div.rounded')
+    .should('exist');
+
+  cy.get('button').contains('Hello').should('exist');
+  cy.get('button').contains('Today').should('exist');
+  cy.get('button')
+    .contains('Hello')
+    .get('svg.text-gray-800')
+    .should('have.class', 'text-gray-800');
+
+  cy.get('#calendar').contains(`${new Date().getFullYear()}`).next().click();
+  cy.get('li').get('.text-gray-600').contains('1').click();
+
+  cy.get('#calendar')
+    .get('button.bg-indigo-700')
+    .get('div.rounded')
+    .should('exist');
```

Comment:
You don’t need to write .should('exist') because .get() has a default assertion and also will wait for element if needed